contents/blog/first-meetup/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Join Us in October for Our First Meetup
+date: 2023-08-14
+author: Michael Robinson
+image: ./image.svg
+banner: ./banner.svg
+description: The first Marquez meetup will be held on 10/5/23 in San Francisco at Astronomer's HQ. 
+---
+
+Join us on Thursday, October 5th, from 5:30-8:30 pm to learn about the present 
+and future of the Marquez project. Meet other members of the community, learn 
+about the project’s goals and fundamental design, and participate in a discussion 
+about the future of the project. Bring your ideas and vision for Marquez!
```

Comment:
I think we should mention something about:
* Brainstorming on data lineage
* The journey from Marquez to OpenLineage adoption (or similar wording i.e. backstory)
* Submitting your first PR (in-person!)
* Stickers!
* Marquez @ Astronomer